spec/src/main/asciidoc/SOAPAttachments.adoc

```diff
@@ -5,10 +5,7 @@
 :sectnums:
 = Jakarta SOAP Attachments Specification, Version 2.0
 
-Copyright (c) 2017, 2020 Oracle and/or its affiliates. All rights reserved.
-
-Oracle and Java are registered trademarks of Oracle and/or its 
-affiliates. Other names may be trademarks of their respective owners. 
+Copyright (c) 2020 Eclipse Foundation.
```

Comment:
This can be removed. The full license is [here](https://github.com/eclipse-ee4j/saaj-api/blob/master/spec/src/main/asciidoc/license-efsl.adoc)